controller/commentsController.js

```diff
@@ -21,4 +21,13 @@ exports.getPostedCommentByArticleId = (req,res,next) =>{
         res.status(201).send(comment)
     })
     .catch(next)
+}
+
+exports.getDeletedCommentByCommentId = (req,res,next) =>{
+    const id = req.params.comment_id
+
+    selectDeletedCommentByCommentId(id).then((deletedComment)=>{
+        res.status(204).send(deletedComment)
```

Comment:
204 for no content won't send back a response body so we don't need to pass the comment into send here, sendStatus() is also an appropriate alternative